mlflow/sagemaker/cli.py

```diff
@@ -26,7 +26,8 @@ def commands():
 @click.option("--region-name", "-r", default="us-west-2", 
               help="Name of the AWS region in which to deploy the application")
 @click.option("--mode", "-md", default=mlflow.sagemaker.DEPLOYMENT_MODE_CREATE, 
-              help="The mode in which to deploy the application")
+        help="The mode in which to deploy the application. Must be one of the modes listed
```

Comment:
travis seems unhappy with this

Reply:
also is it possible to just list the options here directly? something like
```
help="Model deployment mode, one of: %s" % ", ".join(mlflow.sagemaker.DEPLOYMENT_MODES)
```
? then users don't need to look at the source code.

Reply:
Yep, that works. Fixed - Travis should no longer complain.